main.js

```diff
@@ -2,6 +2,7 @@
 const {app, BrowserWindow, BrowserView, Menu} = require('electron')
 const path = require('path')
 const StreamlitServer = require('./python_environment').StreamlitServer
+const portfinder = require('portfinder');
```

Comment:
Nice! I think this is what datasette uses also.

Reply:
Yup, totally just copied from them YET AGAIN :)